CHANGELOG.md

```diff
@@ -13,6 +13,12 @@ _Note: Gaps between patch versions are faulty, broken or test releases._
 
 See [CHANGELOG - 6to5](CHANGELOG-6to5.md) for the pre-4.0.0 version changelog.
 
+## Unreleased
+
+  * **Internal**
+
+    * `babel-types`: Start test suite.
+
```

Comment:
It might turn out to be more trouble than it's worth. (For one thing it might require PRs to be rebased that otherwise wouldn't need to be.) I decided to give it a shot and see how it goes.